### PR TITLE
image-set-view: FIX  image-set image preload data.

### PR DESCRIPTION
**Pull request by ldjebran**
FIXES THEEDGE-2857.

## Type of change

What is it?

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

File: pkg/services/imagesets.go [hunk 1]
```diff
@@ -352,6 +380,13 @@ func (s *ImageSetsService) GetImageSetImageViewByID(imageSetID uint, imageID uin
 		return nil, result.Error
 	}
 
+	// seems gorm will preload all the data from all the images of the image set , that may affect performance if the image-set
+	// has too many images, loading the data one by one is the only to ensure we are loading only the needed one
```
Comment (tomershinhar) [inline pkg/services/imagesets.go:384]: should be "the only way to ensure"

**Review by tomershinhar — COMMENTED** [review]
LGTM
had one tiny comment about the docstring

**Review by tomershinhar — APPROVED** [review]
LGTM

**Review by AaronH88 — APPROVED** [review]
/lgtm

**Pull request merged**